mappings/net/minecraft/phys/BoundingBox.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cmm net/minecraft/util/math/BoundingBox
+CLASS cmm net/minecraft/phys/BoundingBox
```

Comment:
This should be kept to a general `math` package along with any other common vector or math data classes, imo.

Reply:
It really shouldn't. It's the axis-aligned bounding box class, used pretty much entirely for physics.

Reply:
It's a data class for holding and transforming two vectors (min, max), there's no need for it to be associated with physics. Maybe this should be brought up for proper discussion

Reply:
It's not a generic data class, it's specialised.

Reply:
It is, yes. BoundingBox includes raytracing logic, as far as I see.